src/parser/parser.cpp

```diff
@@ -379,10 +377,11 @@ api::Sort Parser::mkUnresolvedTypeConstructor(
 {
   Debug("parser") << "newSortConstructor(P)(" << name << ", " << params.size()
                   << ")" << std::endl;
-  api::Sort unresolved =
-      api::Sort(d_solver,
-                d_solver->getExprManager()->mkSortConstructor(
-                    name, params.size(), ExprManager::SORT_FLAG_PLACEHOLDER));
+  // api::Sort unresolved =
+  //    api::Sort(d_solver,
+  //              d_solver->getExprManager()->mkSortConstructor(
+  //                  name, params.size(), ExprManager::SORT_FLAG_PLACEHOLDER));
```

Comment:
Why are these lines commented out and not removed?

Reply:
I'll remove